src/scripts/main.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+const message = document.createElement('div');
+
+message.classList.add('message');
+
+const badResult = function (content) {
+  message.classList.add('error-message');
+
+  message.textContent = content;
+  body.append(message);
+
+  return message;
+};
+
+const goodResult = function (content) {
+  message.classList.toggle('error-message', false);
+  message.textContent = content;
+  body.append(message);
+
+  return message;
+};
```

Comment:
you need to create the message in this function instead of changing already created element, also try to combine these functions into one